src/autoschedulers/adams2019/ASLog.h

```diff
@@ -31,6 +31,9 @@ class aslog {
     static int aslog_level();
 };
 
+// Conform the given name into a valid C++ identifier (eg for dumping a Func/Var inside a schedule to a header)
+std::string conform_name(const std::string &name, const std::string &prefix="_"); 
```

Comment:
This function doesn't have anything to do with logging, and thus doesn't belong in ASLog.h

Reply:
es, that's true. I placed it here since it was a common header, and deals with formatting text, but it's not actually doing any logging.

Would you propose a new header/source file or some other location?

Reply:
It looks like it's only used in the one source file LoopNest.cpp? I would just put it in there (in an anonymous namespace too).

Reply:
SGTM